src/index.tsx

```diff
@@ -1,14 +1,14 @@
 import { createRoot } from 'react-dom/client';
 import { Root } from './Root';
-import { HashRouter } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 
 import './index.css';
 import 'flowbite';
 
 const container = document.getElementById('root')!;
 const root = createRoot(container);
 root.render(
-  <HashRouter>
+  <BrowserRouter>
```

Comment:
I used `HashRouter` because it is the only type of router that works with GitHub pages, but using the new domain I don't know if the `BrouwserRouter` will work or not :)

Reply:
Google tells me you just need to add a `basename` to the `Router`; I will deploy this to Pages on my own and get back.

Reply:
Let me give a shortcut: after deploying to Github actions try to consult one of the pages directly by typing `your-url/alerts` to the browser.

Reply:
First add `PUBLIC_URL` env to the `Build code 🛠` step in the `Publish Docs` workflow as `/flowbite-react` then pages will work for you

Reply:
Okay I might not seen this before but this can be a solution for the github 404 issue: https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing

Reply:
I think this is a dead end for now... I've remitted the router change from this PR so it can be merged.